docs/concepts/securing-lokomotive-cluster.md

```diff
@@ -33,6 +33,9 @@ default PSPs for general purpose application usage:
     Allowed to workloads in `kube-system` namespace only. This PSP does not restrict any workloads
     from any required permissions.
 
+Many projects provide their own Pod Security Policies tailored to their needs which should be used
```

Comment:
From security perspective, I'm not sure if saying `should` here is what we want. The policies we provide are secure and may work just fine for properly hardened workloads.

```suggestion
Many projects provide their own Pod Security Policies tailored to their needs which can be used
```